### DISCUSSION TITLE
FirebaseDatabase 질문

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/GumamdongAR#3] [Q&A]
CloudAnchorExample 을 돌려보려고 하는데, 
Firebase가 설치 되어 있는데도 불구하고 아래와 같은 에러가 발생해서 진행이 되질 않습니다. 

![Screenshot 2024-10-08 at 12 10 31 AM](https://github.com/user-attachments/assets/f98152a0-ea9e-4229-949b-2303b38f938f)

예제는 구글 ARCore Github에서 최신 버전을 받았는데, 
업데이트를 해도 버전이 갱신이 안되고 ... 삭제도 어디서 하는지 잘 모르겠고, Firebase 재설치도 안되네요. 

혹시 관련해서 같은 예제를 받아서 잘 돌아가는지 확인해주실 수 있나요? 
https://github.com/google-ar/arcore-ios-sdk

**Comment by hemil0102** [comment]
ARCore를 설치하면 생겨나는 Firebase와 달리, 매뉴얼에 다른 경로로 설치하는 것을 제시해서 재시도가 필요

**Comment by jongfeel** [comment]
저도 해보고 피드백 드리겠습니다.

**Reply by hemil0102** [reply]
1. 파이어 베이스 최신을 깔고 ARCore 설치를 시도 했을 경우 파일이 겹쳐서 제대로 설치가 되지 않음. 
2. ARCore를 설치하고 파이어 베이스 최신을 설치할 경우에도 마찬가지로 중복 설치 에러가 발생. 

파베는 처음 사용해보는거라 익숙하지 않은데, 
우선은 클라우드 앵커쪽은 조금 보류하고 Geospatial쪽을 조금더 보고 있습니다.
